### ISSUE TITLE
Cannot issue due to revocation error

### ISSUE THREAD

**Issue by JanLin** [decentralised-dataexchange/aries-playground#27]
On indy ledger performed following curl command got an error when trying to issue in the manual VC creation. Experimenting with turning off revocation but was unsuccessful.

curl --location --request POST 'http://test-center.swagger.localhost/issue-credential/records/a3e5403b-1b54-49b3-ae13-0f479419164c/issue' \
--header 'Content-Type: application/json' \
--data-raw '{
  "comment": "string"
}'

400: Cred def id 4wDz2i3yarKw96TVQrS2S3:3:CL:29:default has no active revocation registry.

test-center.localhost       | aries_cloudagent.revocation.error.RevocationError: URI None/4wDz2i3yarKw96TVQrS2S3:4:4wDz2i3yarKw96TVQrS2S3:3:CL:31:default:CL_ACCUM:ad3a1f45-3c79-4132-add1-e54720b52d45 is not a valid URL

**Comment by lalc** [comment]
Was this done before the connection because status “active”?

**Comment by JanLin** [comment]
Correct. Connection was in active state when using indy ledger. Followed steps in Aries Playground. Below are the states of Issuer and Holder.


Issuer                  | Holder
offer_sent            | request_sent
request_received |  -

**Comment by lalc** [comment]
So what happens after the agents are active?

If you got the agents to active state, when you created the cred-def, please set the support revocation to false as described at: https://docs.igrant.io/ssi/ssi-apg#prepare-public-schema

**Comment by lalc** [comment]
When you created the cred-def, please set the support revovation to false as described at: https://docs.igrant.io/ssi/ssi-apg#prepare-public-schema
